solidity/contracts/LinkEx.sol

```diff
@@ -18,15 +37,68 @@ contract LinkEx is LinkExInterface {
     return historicRate;
   }
 
-  function update(uint256 _rate) external {
+  function removeOracle(address _oracle) external onlyOwner {
+    _removeOracle(_oracle);
+    if (oracles.length > 0) {
+      _update();
+    }
+  }
+
+  function _removeOracle(address _oracle) private {
```

Comment:
I'd prefer to be verbose and have explicit names, instead of denoting private methods(introduced purely for gas savings) with an underscore. Can we call the private method something like `deauthorizeAddress`?